content/en/docs/tasks/security/authz-tcp/index.md

```diff
@@ -105,85 +134,42 @@ Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpa
 * The **Book Reviews** section on the lower right of the page includes an error message **"Ratings service is
   currently unavailable"**.
 
-This is because Istio authorization is "deny by default", which means that you need to explicitly
-define access control policies to grant access to the MongoDB service.
+## Enforce workload-level access control for TCP traffic
 
-{{< tip >}}
-There may be some delays due to caching and other propagation overhead.
-{{< /tip >}}
-
-## Enforcing access control on TCP service
-
-Now let's set up service-level access control using Istio authorization to allow `v2` of `ratings`
-to access the MongoDB service.
-
-Run the following command to apply the authorization policy:
+The command creates a `bookinfo-ratings-v2` policy that grants the access to the MongoDB workload
+at port 27017 for `cluster.local/ns/default/sa/bookinfo-ratings-v2` service account
+that represents the `ratings-v2` workload:
 
 {{< text bash >}}
-$ kubectl apply -f @samples/bookinfo/platform/kube/rbac/mongodb-policy.yaml@
+$ kubectl apply -f - <<EOF
```

Comment:
Please refer to samples in the the repo, see my previous comment on the topic for alternatives.

Reply:
Please see my reply in previous comment.

Reply:
@yangminzhu I can't find the reply on a previous comment. What is the issue? Is it that we are on a code freeze and thus you are unable to update the sample? If that is the case, it would suffice if you file an issue to make sure that the sample gets updated as soon as the code freeze is lifted and the document updated.

Reply:
There are 3 reasons:
1) Allow users to see the policy content when running the command
2) Reduce maintenance burden for consolidating all related yamls in the same task page instead of separating it in a different repo.
3) Align with the existing convention in authentication task page.

With this change, we don't need to worry about the code freeze or any other issues in istio repo at all.